packages/swapper/src/api.ts

```diff
@@ -33,19 +31,96 @@ export type SupportedSellAssetsInput = {
   assetIds: CAIP19[]
 }
 
+export type CommonTradeInput = {
+  sellAsset: Asset
+  buyAsset: Asset
+  sellAmount?: string
+  buyAmount?: string
+  sendMax: boolean
+  sellAssetAccountId: string
+}
+export type GetTradeQuoteInput = CommonTradeInput
+
+export type BuildTradeInput = CommonTradeInput & {
+  buyAssetAccountId: string
+  slippage?: string
+  wallet: HDWallet
+}
+
+interface TradeBase<C extends ChainTypes> {
+  success: boolean // This will go away when we correctly handle errors
+  statusReason: string // This will go away when we correctly handle errors
+  buyAmount: string
+  sellAmount: string
+  feeData: chainAdapters.QuoteFeeData<C>
+  rate: string
+  allowanceContract: string
+  sources: Array<SwapSource>
+  buyAsset: Asset
+  sellAsset: Asset
+  sellAssetAccountId: string
+}
+
+export interface TradeQuote<C extends ChainTypes> extends TradeBase<C> {
+  minimum: string | null
```

Comment:
```suggestion
  minimum?: string
```

can we do this? is this a relic?